<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtChecksum</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qthash.qrc">
    <normaloff>:/icons/qthash.png</normaloff>:/icons/qthash.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QFrame" name="textFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0" colspan="2">
          <widget class="QTextEdit" name="resultTextEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Drag file(s) and Drop to here</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true">QTextEdit{ line-height: 110%; }</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
        <widget class="QLabel" name="empty_placeholder">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>273</y>
           <width>207</width>
           <height>19</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Drag file(s) and Drop to here</string>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Drag file(s) and Drop to here</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy">
          <cstring>searchWidget</cstring>
         </property>
        </widget>
        <zorder>resultTextEdit</zorder>
        <zorder>empty_placeholder</zorder>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="searchWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#searchWidget{
background-color:rgba(0,0,0,0.1);
padding:1px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="searchLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="searchLabel">
           <property name="text">
            <string>Search:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="searchEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>28</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#searchEdit {
margin-top:2px;
margin-bottom:2px;
border:none;
}</string>
           </property>
           <property name="maxLength">
            <number>1024</number>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="prevButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">#prevButton {
background-color:transparent;
border-width: 0px;
      border-style: solid;
      border-color: white;
	 padding:1px;
     margin-left:8px;
}

#prevButton:hover {
background-color: rgba(0,0,0,0.1);
}

#prevButton:pressed {
background-color: rgba(0,0,0,0.2);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="qthash.qrc">
             <normaloff>:/icons/arrow-left.svg</normaloff>:/icons/arrow-left.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">#nextButton {
background-color:transparent;
border-width: 0px;
      border-style: solid;
      border-color: white;
	 padding:1px;
     margin-left:6px;
}

#nextButton:hover {
background-color: rgba(0,0,0,0.1);
}

#nextButton:pressed {
background-color: rgba(0,0,0,0.2);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="qthash.qrc">
             <normaloff>:/icons/arrow-right.svg</normaloff>:/icons/arrow-right.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnHideSearch">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>0</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">#btnHideSearch {
background-color:transparent;
border-width: 0px;
      border-style: solid;
      border-color: white;
	 padding-left:0px;
     padding-right:0px;
     margin:0px;
}

#btnHideSearch:hover {
background-color: rgba(0,0,0,0.1);
}

#btnHideSearch:pressed {
background-color: rgba(0,0,0,0.2);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="qthash.qrc">
             <normaloff>:/icons/close.svg</normaloff>:/icons/close.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="hashAlogrithmBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Hash algorithm </string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QCheckBox" name="crc32CheckBox">
           <property name="text">
            <string>CRC32</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="md5CheckBox">
           <property name="text">
            <string>MD5</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="sha1CheckBox">
           <property name="text">
            <string>SHA1</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QCheckBox" name="sha256CheckBox">
           <property name="text">
            <string>SHA256</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QCheckBox" name="sha512CheckBox">
           <property name="text">
            <string>SHA512</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="sha3_256CheckBox">
           <property name="text">
            <string>SHA3-256</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="sha3_512CheckBox">
           <property name="text">
            <string>SHA3-512</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="keccak256CheckBox">
           <property name="text">
            <string>KECCAK-256</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QCheckBox" name="keccak512CheckBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>KECCAK-512</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>16</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>16</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
            <property name="spacing">
             <number>16</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QPushButton" name="browseButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Browse ...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>102</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="saveButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>102</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="copyButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>102</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Copy</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="abortButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimumSize">
               <size>
                <width>102</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Abort</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="spacing">
             <number>8</number>
            </property>
            <item row="1" column="1">
             <widget class="QProgressBar" name="fileProgressBar">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelTotal">
              <property name="text">
               <string>Total progress:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QProgressBar" name="totalProgressBar">
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelFile">
              <property name="text">
               <string>File progress:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="optionsButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>52</width>
            <height>52</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="qthash.qrc">
            <normaloff>:/icons/options.svg</normaloff>:/icons/options.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>40</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="checkboxLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qthash.qrc"/>
 </resources>
 <connections/>
</ui>
